cfgov/jinja2/v1/_includes/macros/time.html

```diff
@@ -41,3 +41,20 @@
     {% endif %}
 </span>
 {% endmacro %}
+
+{% macro render_date_no_markup(datetime,
+                show_value_for={'date':true, 'time':true, 'timezone':true}) %}
+    {# TODO: Replace this with a custom Jinja method that uses
+             https://dateutil.readthedocs.io/ #}
+    {# Displays the date in the format: MMM. DD, YYYY #}
+    {% set format_month = '{dt:%b}'.format(dt = datetime) %}
+    {% if format_month == 'May' %}
+        {# May doesn't have a period. #}
+        {{ '{dt:%b} {dt.day}, {dt.year}'.format(dt = datetime) }}
+    {% elif format_month == 'Sep' %}
+        {# September is four letters. #}
+        {{ 'Sept. {dt.day}, {dt.year}'.format(dt = datetime) }}
+    {% else %}
+        {{ '{dt:%b}. {dt.day}, {dt.year}'.format(dt = datetime) }}
+    {% endif %}
```

Comment:
I spent some time thinking about this and looking at possible alternatives. I couldn't find a Python library that has built-in support for formatting months like this (conditionality around when a month is abbreviated and to what character-number).  

At the very least, I don't think this should be in the template itself, it should be a filter or something similar that can be called from any template (particularly if this is the standardized way to refer to dates) and can be properly unittested. 

It would also simplify the logic greatly to [use a mapping like Python's build-in datetime library of month number to abbreviation](https://github.com/python/cpython/blob/master/Lib/datetime.py#L144-L146) instead of ifs/elses.